src/main/resources/org/threeten/extra/scale/LeapSeconds.txt

```diff
@@ -25,3 +25,4 @@
 2005-12-31 33
 2008-12-31 34
 2012-06-30 35
+2015-06-30 36
```

Comment:
You shouldn't need to add this yourself.  Once the review is done you'll be rebasing to master (possibly via a second pull request) to squash you history anyways.  That will pull in any changes.